tests/lit-tests/ispc-intrinsics.ispc

```diff
@@ -0,0 +1,82 @@
+// RUN: %{ispc} %s --nowrap --target=host --emit-llvm-text --enable-llvm-intrinsics -o - 2>&1 | FileCheck %s
```

Comment:
Why do we want to test them for host width only, but not for all width that intrinsics supposed to cover? I think we want to ensure that all relevant target width are tested.